bundles/org.openhab.binding.saicismart/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.saicismart</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: SAICiSMART Binding</name>
+  <properties>
+    <bnd.importpackage>org.brotli.dec;resolution:=optional,org.conscrypt;resolution:=optional,org.glassfish.jersey.internal.spi;resolution:=optional,org.graalvm.nativeimage.hosted;resolution:=optional,org.joda.time;resolution:=optional,org.joda.time.base;resolution:=optional,org.joda.time.format;resolution:=optional,org.springframework.core;resolution:=optional,org.springframework.http;resolution:=optional,org.springframework.http.converter;resolution:=optional,org.springframework.web;resolution:=optional,org.springframework.web.bind.support;resolution:=optional,org.springframework.web.context.request;resolution:=optional,org.springframework.web.method.support;resolution:=optional,org.springframework.web.servlet.mvc.method.annotation;resolution:=optional,picocli;resolution:=optional</bnd.importpackage>
+  </properties>
+
+  <dependencies>
```

Comment:
I'm no expert, but pretty sure about all these importpackes/dependencies should be removed when creating a PR. You probably used it to create a jar for stable versions ?

Reply:
I am not sure I understand this dependency injection but I think there are a lot of class not found runtime errors without out it.  I will investigate more.

Reply:
I am struggling with the dependency management. I read this:
https://www.openhab.org/docs/configuration/addons.html#through-manually-provided-add-ons

So I guess the dependency management is different for my testing version (installed with jar in folder) and the bundle installed version.  I am not sure how to accurately install my binding with the same dependency management that the finished version would have?  Without this it is hard to clean up all the dependencies.

Reply:
I got the install with the openhab console working and I have reduces the dependencies.

Reply:
Why does this binding rely on these packages? And if so why are they optional?

Reply:
Not sure.  These are the saic java lib (https://github.com/SAIC-iSmart-API/saic-java-client) dependencies that the binding depends on.  If they are not optional the bundle will not start.  I do not fully understand how the bnd.importpackages works.

Reply:
Ok, as they are needed, they should be listed in the notice file.